examples/run/alpaka/seeding_example_alpaka.cpp

```diff
@@ -63,8 +67,15 @@ int seq_run(const traccc::opts::track_seeding& seeding_opts,
     using Idx = uint32_t;
 
     using Acc = ::alpaka::ExampleDefaultAcc<Dim, Idx>;
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+    ::sycl::queue q;
+    vecmem::sycl::queue_wrapper qw{&q};
+    traccc::alpaka::vecmem::host_device_types<
+        ::alpaka::trait::AccToTag<Acc>::type>::device_copy copy(qw);
+#else
     traccc::alpaka::vecmem::host_device_types<
         ::alpaka::trait::AccToTag<Acc>::type>::device_copy copy;
+#endif
     traccc::alpaka::vecmem::host_device_types<
         ::alpaka::trait::AccToTag<Acc>::type>::host_memory_resource host_mr;
```

Comment:
Note that it's not just the copy object, but all memory resources also need to be constructed explicitly on top of the same `::sycl::queue` object. Otherwise all sorts of weird behaviours can happen. 🤔 (Though not necessarily using all backends. And with some backends, only if you have multiple possible devices available.)

Reply:
OK, this makes a lot of sense. I guess they should also be taking the queue as argument?

Reply:
I added queue_wrapper to all the memory_resources.